### PR TITLE
[Security][Translation] fixes #14584

### LINKED ISSUE #14584: [Security][Translation] “credentials” doesn't translate to “droits” in french
In files **security.fr.xlf**, “credentials” is translated to “droits” in sentences

- Authentication credentials could not be found.
- Invalid credentials.
- Credentials have expired.

but it translates to “identifiants”, so translations should be

- *Les identifiants d'authentification n'ont pas pu être trouvés.*
- *Identifiants invalides.*
- *Les identifiants ont expiré.* (“*expiré*” doesn't need a “s”.)

### PR DESCRIPTION
**Pull request by MatTheCat**
| Q | A |
| --- | --- |
| Fixed tickets | #14584 |
| License | MIT |

Some french translations are wrong  in the security component.
As #14587 has been closed here's my fix.

### CODE REVIEW

**Comment by fabpot** [comment]
Need some feedback from French people. Is it better like this?

**Comment by msanichanh** [comment]
Yes it is !

**Comment by viviengaetan** [comment]
Yes it's written in French now ;)

**Comment by Romauge** [comment]
I confirm, it is ok !

**Comment by 94noni** [comment]
It's better yes

File: src/Symfony/Component/Security/Resources/translations/security.fr.xlf [hunk 1]
```diff
@@ -8,7 +8,7 @@
             </trans-unit>
             <trans-unit id="2">
                 <source>Authentication credentials could not be found.</source>
-                <target>Les droits d'authentification n'ont pas pu être trouvés.</target>
+                <target>Les identifiants d'authentification n'ont pas pu être trouvés.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Security/Resources/translations/security.fr.xlf]: Perso identifiant me semble redondant avec authentification, je mettrais bien `Les identifiants n'ont pu être trouvés.` ici.

**Comment by nicolas-grekas** [comment]
:+1: (with or without my note above)

File: src/Symfony/Component/Security/Resources/translations/security.fr.xlf (hunk 1 above)
Comment (MatTheCat) [inline src/Symfony/Component/Security/Resources/translations/security.fr.xlf]: J'ai préféré coller à la traduction ; comme la distinction n'est faite qu'ici je l'ai laissée.

**Comment by fabpot** [comment]
Thank you @MatTheCat.

**Pull request merged**